app/models/book_sugestion.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class BookSugestion < ApplicationRecord
+  validates :author, :title, :link, presence: true
+
+  has_one :user, dependent: :destroy
```

Comment:
Are you sure you want a  `dependent: :destroy`? 
[Here](https://guides.rubyonrails.org/association_basics.html) some help.